catalog/tests/dags/providers/provider_api_scripts/resources/smk/expected_image_data_legacy.json

```diff
@@ -1,12 +1,12 @@
 {
   "filesize": 11784886,
   "height": 1059,
-  "url": "https://api.smk.dk/api/v1/thumbnail/52f00edc-936e-42a7-950b-d0cd0df3864b.jpg",
-  "thumbnail_url": "https://api.smk.dk/api/v1/thumbnail/52f00edc-936e-42a7-950b-d0cd0df3864b.jpg",
+  "url": "https://api.smk.dk/api/v1/thumbnail/KKSgb5743%20verso.jpg",
+  "thumbnail_url": "https://api.smk.dk/api/v1/thumbnail/KKSgb5743%20verso.jpg",
```

Comment:
Instead of modifying the test data, can we create an extra test case for spaces in URLs? Should the same apply to the direct URL?